app/sidebar.py

```diff
@@ -28,6 +29,14 @@
         html.H6(
             id="data_description", style={"textAlign": "left", "color": colors["text"]}
         ),
+        html.H6(
+            html.A(
+                id="data_citation",
+                href="https://pubs.acs.org/doi/abs/10.1021/ic020647f",
+                target="_blank",
```

Comment:
You should not hard code the citation. This way, no matter where the data comes from, the link will always be ``https://pubs.acs.org/doi/abs/10.1021/ic020647f``. One solution would be to have the source address in the JSON file as the value of the `citation` key, which may change from file to file. You can then use the this value to update the `href` of the `html.A`.